source/screens/caseScreens/CaseSummary.js

```diff
@@ -479,18 +479,28 @@ const CaseSummary = (props) => {
                         )}
 
                         <Card.DetailsTitle type="h5">Reducering</Card.DetailsTitle>
-
                         {calculation?.reductions?.reduction ? (
                           <>
-                            <Card.CalculationRow>
-                              <Card.CalculationRowCell>
-                                <Text>{calculation.reductions.reduction}</Text>
-                              </Card.CalculationRowCell>
-                              <Card.CalculationRowCell />
-                              <Card.CalculationRowCell>
-                                <Text>{formatAmount(calculation.reductions.reduction.amount)}</Text>
-                              </Card.CalculationRowCell>
-                            </Card.CalculationRow>
+                            {convertDataToArray(calculation?.reductions?.reduction).map(
+                              (reduction, index) => (
+                                <Card.CalculationRow key={`${index}-${reduction.type}`}>
+                                  <Card.CalculationRowCell>
+                                    <Text>{reduction?.type}</Text>
+                                  </Card.CalculationRowCell>
+                                  <Card.CalculationRowCell>
+                                    <Text>
+                                      {reduction?.days} {reduction?.days > 1 ? 'dagar' : 'dag'}
+                                    </Text>
+                                  </Card.CalculationRowCell>
+                                  <Card.CalculationRowCell>
+                                    <Text>{reduction?.note}</Text>
+                                  </Card.CalculationRowCell>
+                                  <Card.CalculationRowCell>
+                                    <Text>{formatAmount(reduction.amount)}</Text>
+                                  </Card.CalculationRowCell>
+                                </Card.CalculationRow>
+                              )
+                            )}
                           </>
```

Comment:
So there is one concern for me here and it lies in what attributes we expect reductions to have in order to render this section. What is required in order to render the reduction information in regards for the attributes amount, note, type and days? 

Either way i would argue that checking the value before rendering the element is a better approach, due to that we can avoid unnecessary rendering of Card.CalculationRowCell components.

But in regards for the rest of this massive component, i don't know how big of a value this is to change.

Reply:
Agree, if some values are optional then something like

```javascript
{reduction.type && (
  <Card.CalculationRowCell>
    <Text>{reduction.type}</Text>
  </Card.CalculationRowCell>
)}
```

could be used instead, however the entire component could use some bigger refactoring, perhaps outside the scope of this PR.

Reply:
Agree, we should add this refactoring to the new backlog.